types/tx2/package.json

```diff
@@ -0,0 +1,6 @@
+{
```

Comment:
I don't think this file is actually needed; there are only two declaration packages in the whole repo which explicitly depend on these node types.

Reply:
I'm happy to remove it!
Do I just keep the `/// <reference types="node" />` ?

Reply:
Yes, that's correct.